plugins/item_tasks/server/rest.py

```diff
@@ -40,6 +41,30 @@ def listTasks(self, limit, offset, sort, params):
             cursor, self.getCurrentUser(), level=AccessType.READ, limit=limit, offset=offset,
             flags=constants.ACCESS_FLAG_EXECUTE_TASK))
 
+    @access.public
+    @autoDescribeRoute(
+        Description('Search for tasks that can be executed in the system.')
+        .param('q', 'The search query.')
+        .pagingParams(defaultSort=None, defaultLimit=20)
+    )
+    def search(self, q, limit, offset, params):
+        user = self.getCurrentUser()
+        itemModel = self.model('item')
+
+        filters = {
+            'meta.isItemTask': {'$exists': True}
```

Comment:
This plugin ignores the value of `isItemTask`, probably as an optimization. This is actually documented: https://girder.readthedocs.io/en/latest/plugins.html#item-tasks.

Note that this same query is performed in `listTasks()`.